py3-cloudpickle.yaml

```diff
@@ -0,0 +1,41 @@
+# Generated from https://pypi.org/project/cloudpickle/
```

Comment:
not sure if we need these at the top of each either, not a blocker but feels like we we're tidying up our configs and making them more consistent.  So maybe drop all the `# Generated from...`

Reply:
removed!